amgut/db/patches/0032.sql

```diff
@@ -0,0 +1,369 @@
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Samoa', 189);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'San Marino', 190);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Sao Tome and Principe', 191);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Saudi Arabia', 192);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Senegal', 193);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Serbia', 194);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Seychelles', 195);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Sierra Leone', 196);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Singapore', 197);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Slovakia', 198);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Slovenia', 199);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Solomon Islands', 200);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Somalia', 201);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'South Africa', 202);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'South Georgia and The South Sandwich Islands', 203);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Spain', 204);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Sri Lanka', 205);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Sudan', 206);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Suriname', 207);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Svalbard and Jan Mayen', 208);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Swaziland', 209);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Sweden', 210);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Switzerland', 211);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Syrian Arab Republic', 212);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Taiwan, Province of China', 213);
```

Comment:
That's what it is in all other questions. Not sure if we want to fix this or not.